flink-staging/flink-gelly/src/main/java/org/apache/flink/graph/library/SingleSourceShortestPaths.java

```diff
@@ -77,7 +77,7 @@ public Double map(Vertex<K, Double> value) {
 			extends VertexUpdateFunction<K, Double, Double> {
 
```

Comment:
I really like what the examples look like now! Much better having the Vertex object instead of separated key-value ^^